lib/grandpa/grandpa_test.go

```diff
@@ -51,9 +50,11 @@ func NewMockDigestHandler() *mocks.DigestHandler {
 	return m
 }
 
+//go:generate mockgen -source=../../dot/telemetry/telemetry.go -destination=mock_telemetry_test.go -package $GOPACKAGE Client
```

Comment:
Don't use `-source`, just the Go module name `github.com/ChainSafe/gossamer/dot/telemetry`

```suggestion
//go:generate mockgen -destination=mock_telemetry_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/dot/telemetry Client
```

Same for other usage of `-source`